minikube-operator-example/config/300-serving-v1alpha1-knativeserving-crd.yaml

```diff
@@ -0,0 +1,153 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Are these symlinks? I know the deployment one can't be a symlink because there's a diff for image field.

Reply:
Working on replacing with symlinks

Reply:
It seems like ko doesn't handle symlinks with parent directory in path correctly, so I just copy & paste the yamls